docs/configuring-socorro.rst

```diff
@@ -3,271 +3,138 @@
 Configure and run Socorro
 =========================
 
-Initial setup
--------------
-
-Socorro provides a setup script which attempts to initialize everything::
-
-    sudo setup-socorro.sh
-
-Configuring Socorro
--------------------
+Storing configuration in Consul
+-------------------------------
 
 Socorro uses a distributed configuration service called Consul to hold
 configuration - https://consul.io/intro/getting-started/install.html
 
 Consul must be running in order for Socorro apps to start up and access
 their configuration. Normally you want to run a cluster (see the docs above)
-but to start in a single node configuration::
+but to start in a single node configuration add this to
+/etc/consul/server.json::
 
-  sudo consul agent -server -bootstrap-expect 1 -data-dir /var/lib/consul
+  {
```

Comment:
This block assumes that the Consul package was installed from our repo (and, thus, includes `/etc/consul/common.json`).  Given that this is only true if the user is using our repo - the suggestion for which occurs in `production-install.rst` *only* - I don't know if the assumption in question is a safe one.

In other words, either we mandate that our repo be used from the get-go, or we must assume that it isn't being used at all and write our documentation accordingly.  In the latter case, the previous revision (command-line invocation) is the only safe possibility.